docking.js

```diff
@@ -2028,7 +2036,7 @@ var DockManager = class DashToDock_DockManager {
                 delay: STARTUP_ANIMATION_TIME,
                 duration: STARTUP_ANIMATION_TIME,
                 mode: Clutter.AnimationMode.EASE_OUT_QUAD,
-                ...mainDockProperties,
+                onStopped: newCallback,
```

Comment:
Thus here can be:

```suggestion
                onStopped: () => {},
                ...mainDockProperties,
```

Reply:
Oh, actually this is not needed at all, don't think we need to call anything at all, so just keep `...mainDockProperties`.

Reply:
Done.